contracts/cw20-base/src/contract.rs

```diff
@@ -304,8 +307,17 @@ pub fn execute_mint(
         return Err(ContractError::InvalidZeroAmount {});
     }
 
-    let mut config = TOKEN_INFO.load(deps.storage)?;
-    if config.mint.is_none() || config.mint.as_ref().unwrap().minter != info.sender {
+    let mut config = TOKEN_INFO
+        .may_load(deps.storage)?
+        .ok_or(ContractError::Unauthorized {})?;
+
+    if config
```

Comment:
this is nicer than previous code. thnaks

Reply:
It is just because of style, yes. If there are no more concerns, I think we can merge this as it is.